packages/bpk-component-card-button/src/BpkSaveButton.tsx

```diff
@@ -0,0 +1,116 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { MouseEvent } from 'react';
+import { useState } from 'react';
+
+import BpkButtonV2 from '../../bpk-component-button/src/BpkButtonV2/BpkButton';
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
+import BpkHeartIcon from '../../bpk-component-icon/lg/heart';
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
+import BpkHeartOutlineIcon from '../../bpk-component-icon/lg/heart--outline';
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
+import BpkHeartIconSm from '../../bpk-component-icon/sm/heart';
+// @ts-expect-error Untyped import. See `decisions/imports-ts-suppressions.md`.
+import BpkHeartOutlineIconSm from '../../bpk-component-icon/sm/heart--outline';
+import { withLargeButtonAlignment } from '../../bpk-component-icon/index';
+import { cssModules } from '../../bpk-react-utils/index';
+
+import STYLES from './BpkSaveButton.module.scss';
+
+export const SIZE_TYPES = {
+  default: 'default',
+  small: 'small',
+} as const;
+
+export type SizeType = (typeof SIZE_TYPES)[keyof typeof SIZE_TYPES];
```

Comment:
Do we need to add a `Type` suffix to the type name? Not sure backpack standards but it feels a bit redundant

Reply:
It seems that adding Type as a suffix in backpack is ok, e.g. [here](https://github.com/Skyscanner/backpack/blob/main/packages/bpk-component-button/src/BpkButtonV2/common-types.tsx#L33), so I added it